pom.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.cloud.runtimes</groupId>
+  <artifactId>tomcat-parent</artifactId>
+  <version>0.1.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+  <name>Google Tomcat Runtime Parent</name>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <maven.build.timestamp.format>yyyy-MM-dd-HH-mm</maven.build.timestamp.format>
+    <gcloud.api.version>1.0.1</gcloud.api.version>
+    <tomcat9.minor.version>0</tomcat9.minor.version>
+    <tomcat9.dot.version>0</tomcat9.dot.version>
+    <tomcat9.version>9.${tomcat9.minor.version}.${tomcat9.dot.version}.M21</tomcat9.version>
```

Comment:
It's a big question, but I think we want to start with tomcat 8 (8.5). Tomcat 9 doesn't even have a release date yet.